controllers/release/release_suite_test.go

```diff
@@ -34,6 +35,7 @@ import (
 
 	appstudioshared "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
 	appstudiov1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	appstudiotest "github.com/redhat-appstudio/release-service/test"
```

Comment:
I think `releasetest` could've been a better name for it, but because of the line before I think we keep it as is for consistency's sake. I think because there is an appstudio team, my brain trips up on it thinking appstudio is not us